pytest_pulp_cli/__init__.py

```diff
@@ -17,7 +17,7 @@ class ScriptFile(pytest.File):
     obj = None
 
     def collect(self):
-        name = self.path.name[5:]
+        name = self.path.name[5:][:-3]
```

Comment:
I don't think I understand what this change is doing? Can you explain? (and maybe add a comment so I won't ask you the same question in a few months :) )

Reply:
Technically this is not even related. But it was bugging me.

```suggestion
        # Extract the name between "test_" and ".sh".
        name = self.path.name[5:][:-3]
```